Tests/LibCore/TestLibCoreMimeType.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2025, Andreas Kling <andreas@ladybird.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TestLibCoreMimeType.h"
+#include <AK/Vector.h>
+#include <LibCore/MimeData.h>
+#include <LibTest/TestCase.h>
+
+
+void check_filename_mimetype(AK::Vector<StringView> const& filepaths, StringView const& expected_mimeType)
```

Comment:
casing on the second parameter here is a bit off. `expected_mime_type` is more in line with project style.


No need to pass StringView by const ref, it's the size of two pointers (well, pointer + size).

Prefixing Vector with AK is not necessary or in-line with style either.

Reply:
Sounds good. I'll fix those.